cmds/core/ls/ls_test.go

```diff
@@ -83,15 +90,40 @@ func TestLs(t *testing.T) {
 
 	// Create an empty directory.
 	testDir := filepath.Join(tmpDir, "testDir")
-	os.Mkdir(testDir, 0o700)
+	err := os.Mkdir(testDir, 0o700)
+	if err != nil {
+		t.Fatal(err)
+	}
 
 	// Create some files.
-	os.Create(filepath.Join(testDir, "f1"))
-	os.Create(filepath.Join(testDir, "f2"))
-	os.Create(filepath.Join(testDir, "f3\nline 2"))
-	os.Create(filepath.Join(testDir, ".f4"))
-	os.Mkdir(filepath.Join(testDir, "d1"), 0o740)
-	os.Create(filepath.Join(testDir, "d1/f4"))
+	_, err = os.Create(filepath.Join(testDir, "f1"))
```

Comment:
This can be simplified to a for loop:

```
if _, err := os.Mkdir(filepath.Join(testDir, "d1"), 0o740) {
    t.Fatal(err)
}
for _, filename := range []string{"f2", "f3\nline 2", ".f4", "d1/f4"} {
    if _, err := os.Create(filepath.Join(testDir, filename)); err != nil {
        t.Fatal(err)
    }
}
```